dspace-spring-rest/src/main/java/org/dspace/app/rest/model/patch/LateObjectEvaluator.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model.patch;
+
+
+/**
+ * <p>
+ * Strategy interface for resolving values from an operation definition.
+ * </p>
+ *
+ * Based on {@link org.springframework.data.rest.webmvc.json.patch.LateObjectEvaluator}
+ *
+ * @author Luigi Andrea Pascarelli (luigiandrea.pascarelli at 4science.it)
+ */
+public interface LateObjectEvaluator {
```

Comment:
@KevinVdV : This change is required to allow us to upgrade to the latest version of Spring Boot. In more recent versions of Spring Boot, the Spring version of `LateObjectEvaluator` is no longer public, and you are required to implement your own version.  

So, you are correct, creating our own version of this interface looks odd, but there's a reason for it.  Currently we are "locked" to Spring Boot version 1.4.x.  More information on this can be found in this JIRA ticket: https://jira.duraspace.org/browse/DS-3802